h2o-bindings/bin/custom/python/gen_gam.py

```diff
@@ -117,3 +117,459 @@ def get_gam_knot_column_names(self):
+>>> h2o_model.coef()
+""",
+    spline_orders="""
+>>> import h2o
+>>> from h2o.estimators.gam import H2OGeneralizedAdditiveEstimator
+>>> h2o.init()
+>>> knots1 = [-1.99905699, -0.98143075, 0.02599159, 1.00770987, 1.99942290]
+>>> frameKnots1 = h2o.H2OFrame(python_obj=knots1)
+>>> knots2 = [-1.999821861, -1.005257990, -0.006716042, 1.002197392, 1.999073589]
+>>> frameKnots2 = h2o.H2OFrame(python_obj=knots2)
+>>> knots3 = [-1.999675688, -0.979893796, 0.007573327, 1.011437347, 1.999611676]
+>>> frameKnots3 = h2o.H2OFrame(python_obj=knots3)
+>>> h2o_data = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/glm_test/multinomial_10_classes_10_cols_10000_Rows_train.csv")
+>>> h2o_data["C1"] = h2o_data["C1"].asfactor()
+>>> h2o_data["C2"] = h2o_data["C2"].asfactor()
+>>> h2o_data["C11"] = h2o_data["C11"].asfactor()
+>>> train, test = h2o_data.split_frame(ratios = [.8])
+>>> y = "C11"
+>>> x = ["C1","C2"]
+>>> numKnots = [5,5,5]
+>>> h2o_model = H2OGeneralizedAdditiveEstimator(family='multinomial',
+...                                             gam_columns=["C6","C7","C8"],
+...                                             scale=[1,1,1],
+...                                             num_knots=numKnots,
+...                                             knot_ids=[frameKnots1.key, frameKnots2.key, frameKnots3.key],
+...                                             spline_orders=None)
+>>> h2o_model.train(x=x, y=y, training_frame=train)
+>>> h2o_model.coef()
+""",
+    standardize_tp_gam_cols="""
+>>> import h2o
+>>> from h2o.estimators.gam import H2OGeneralizedAdditiveEstimator
+>>> h2o.init()
+>>> knots1 = [-1.99905699, -0.98143075, 0.02599159, 1.00770987, 1.99942290]
+>>> frameKnots1 = h2o.H2OFrame(python_obj=knots1)
+>>> knots2 = [-1.999821861, -1.005257990, -0.006716042, 1.002197392, 1.999073589]
+>>> frameKnots2 = h2o.H2OFrame(python_obj=knots2)
+>>> knots3 = [-1.999675688, -0.979893796, 0.007573327, 1.011437347, 1.999611676]
+>>> frameKnots3 = h2o.H2OFrame(python_obj=knots3)
+>>> h2o_data = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/glm_test/multinomial_10_classes_10_cols_10000_Rows_train.csv")
+>>> h2o_data["C1"] = h2o_data["C1"].asfactor()
+>>> h2o_data["C2"] = h2o_data["C2"].asfactor()
+>>> h2o_data["C11"] = h2o_data["C11"].asfactor()
+>>> train, test = h2o_data.split_frame(ratios = [.8])
+>>> y = "C11"
+>>> x = ["C1","C2"]
+>>> numKnots = [5,5,5]
+>>> h2o_model = H2OGeneralizedAdditiveEstimator(family='multinomial',
+...                                             gam_columns=["C6","C7","C8"],
+...                                             scale=[1,1,1],
+...                                             num_knots=numKnots,
+...                                             knot_ids=[frameKnots1.key, frameKnots2.key, frameKnots3.key],
+...                                             standardize_tp_gam_cols=False)
+>>> h2o_model.train(x=x, y=y, training_frame=train)
+>>> h2o_model.coef()
+""",
```

Comment:
Please replace lines 441 to 465 with:

>>> import h2o
 >>> from h2o.estimators.gam import H2OGeneralizedAdditiveEstimator
 >>> h2o.init()
 >>> h2o_data = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/glm_test/binomial_20_cols_10KRows.csv")
 >>>  y="C21"
 >>> x = ["C19","C20"]
 >>> h2o_model = H2OGeneralizedAdditiveEstimator(family=‘gaussian’, gam_columns=["C16","C17","C18”], bs=[1,1,1], standardize_tp_gam_cols=True)
 >>> h2o_model.train(x=x, y=y, training_frame=h2o_data)
 >>> h2o_model.coef()